includes/settings/class-wcpdf-settings-general.php

```diff
@@ -170,6 +170,32 @@ public function init_settings() {
 					'translatable'	=> true,
 				)
 			),
+			array(
+				'type'     => 'setting',
+				'id'       => 'vat_number',
+				'title'    => __( 'Shop VAT Number', 'woocommerce-pdf-invoices-packing-slips' ),
+				'callback' => 'text_input',
+				'section'  => 'general_settings',
+				'args'     => array(
+					'option_name'  => $option_name,
+					'id'           => 'vat_number',
+					'translatable' => true,
+					'description'  => __( 'Required by the UBL format.', 'woocommerce-pdf-invoices-packing-slips' ),
+				)
+			),
+			array(
+				'type'     => 'setting',
+				'id'       => 'coc_number',
+				'title'    => __( 'Shop Chamber of Commerce Number', 'woocommerce-pdf-invoices-packing-slips' ),
+				'callback' => 'text_input',
+				'section'  => 'general_settings',
+				'args'     => array(
+					'option_name'  => $option_name,
+					'id'           => 'coc_number',
+					'translatable' => true,
+					'description'  => __( 'Required by the UBL format.', 'woocommerce-pdf-invoices-packing-slips' ),
+				)
+			),
```

Comment:
I think these settings should only display in 'General' when the UBL document is enabled. Also I would change the description to: `Required for UBL file generation.`

Reply:
I wouldn't do that, because the code is prepared to have UBL for other documents. If we go that way we should also disable the UBL tab too, which I don't think we should do.

Of course it's doable to hide them, but I see this setting like the `Extra field 1`, `Extra field 2` and `Extra field 3`, which are being displayed even when the Premium Templates is not installed. Beside that, I'm seeing customers using the VAT number field to display in the PDF as well, which can be also added there easily.

Reply:
I would propose to remove the `Required by the UBL format.` description from these settings. And add an admin notice when a UBL document is enabled but certain required settings have no value yet. Something like the following:

`You've enabled UBL format for invoices, but some essential details are missing. Please ensure you've added your VAT and CoC numbers in the General settings [link]. Also, specify your tax rates in the UBL settings [link].`

Also I think we need to create a documentation page for setting up the UBL format. We can link to it from the UBL tab and also from the UBL settings in the Documents tab.

Reply:
I agree, seems a good way to handle it.

@YordanSoares can you create the issue/doc page for the UBL?